From 4fc1d8d05dd39df0ad2a8ed539438f8396d6804a Mon Sep 17 00:00:00 2001
From: QiuJi <qiuji@iscas.ac.cn>
Date: Mon, 22 Nov 2021 17:09:07 +0800
Subject: [PATCH] [riscv64] Re-enable
 test/inspector/debugger/wasm-gc-breakpoints.js

Change-Id: I7755e943674ec4128b0e5869e9ff5ea735efd162
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3295309
Auto-Submit: ji qiu <qiuji@iscas.ac.cn>
Commit-Queue: Kim-Anh Tran <kimanh@chromium.org>
Reviewed-by: Yahan Lu <yahan@iscas.ac.cn>
Reviewed-by: Kim-Anh Tran <kimanh@chromium.org>
Cr-Commit-Position: refs/heads/main@{#78036}
---
 test/inspector/inspector.status | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/test/inspector/inspector.status b/test/inspector/inspector.status
index ffb81d88d00..c74a09a3dee 100644
--- a/test/inspector/inspector.status
+++ b/test/inspector/inspector.status
@@ -125,10 +125,6 @@
 
 }],  # 'arch == riscv64 or arch == loong64'
 
-['arch == riscv64 and variant == stress_incremental_marking', {
-  'debugger/wasm-gc-breakpoints': [SKIP]
-}],  # 'arch == riscv64'
-
 ################################################################################
 ['variant == stress_snapshot', {
   '*': [SKIP],  # only relevant for mjsunit tests.
-- 
2.35.1

